README.md

```diff
@@ -229,6 +229,7 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 
 ## CSS
 
+- :book: [CSS free Course](https://www.w3schools.com/css/)
```

Comment:
Change the title to "CSS Tutorial on W3Schools" and replace the emoji with `:books:`.